neuralforecast/core.py

```diff
@@ -59,6 +60,11 @@
 )
 
 # %% ../nbs/core.ipynb 5
+# this disables warnings about the number of workers in the dataloaders
+# which the user can't control
+pl.disable_possible_user_warnings()
```

Comment:
I think this addition requires to bump the [required version](https://github.com/Nixtla/neuralforecast/blob/e0ecff827338af875acba76cf9e97e936ac56849/settings.ini#L18) of `pytorch-lightning` from 2.0.0 to 2.1.0, because the function `disable_possible_user_warnings` was only introduced in 2.1.0; see [the according PR](https://github.com/Lightning-AI/pytorch-lightning/commit/e02bb391af61eb02cf104ae032549b20243ec8f9) for context.
Many thanks for the great package 🙌

Reply:
Thanks! I recently saw this as well and thought about changing that line to what the function does
https://github.com/Lightning-AI/pytorch-lightning/blob/2064887b12dd934a5f9a2bf45897f29e3bfc74d1/src/lightning/fabric/utilities/warnings.py#L37
since that warning type has been there longer. Would you like to contribute that?

Reply:
... happy to contribute this minor issue [in this PR](https://github.com/Nixtla/neuralforecast/pull/1081) . Note that this is my first contribution - I hope all is fine